misc/ls-extensions/modules/partial-parser/src/main/java/io.ballerina.parsers/PartialParserService.java

```diff
@@ -39,24 +45,28 @@
 @JavaSPIService("org.ballerinalang.langserver.commons.service.spi.ExtendedLanguageServerService")
 @JsonSegment("partialParser")
 public class PartialParserService implements ExtendedLanguageServerService {
+    private LSClientLogger clientLogger;
```

Comment:
I think you wanted ls core to get the client logger. Instead, you should override the `connect(ExtendedLanguageClient client)` method and get the `ExtendedLanguageClient`. Then you can use it to log messages when required.

Reply:
Tried this approach and encountered [this](https://github.com/wso2-enterprise/internal-support-ballerina/issues/67).
Will address this once that issue is fixed.